src/mailroom.py

```diff
@@ -0,0 +1,121 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+from __future__ import print_function
+try:
+    input = raw_input
+except NameError:
+    pass
+
+donor_list = {
+    ('lessor', 'crystal'): [10, 40, 25],
+    ('harrison', 'mike'): [30, 40, 15],
+    ('smith', 'john'): [10, 25],
+    ('python', 'monty'): [42],
+}
+
+
+def sort_donors(lst):
+    """Turn donors into list and sorts in decending order."""
+    sorted_donors = sorted(donor_list, key=lambda x: sum(donor_list[x]), reverse=True)
+    new_list = []
+    for k in sorted_donors:
+        avg = float((sum(donor_list[k])) / len(donor_list[k]))
```

Comment:
The behavior of this operator (`/`) is different in Python 2 and Python 3.  Given how you've structured this, will you ever end up with a decimal value other than `0`?